inference-engine/tests/functional/plugin/cpu/single_layer_tests/depth_to_space.cpp

```diff
@@ -114,42 +150,45 @@ const std::vector<CPUSpecificParams> CPUParamsBS2_4D = {
         cpuParams_nhwc_ref,
 };
 
-const auto depthToSpaceBS2_4DParams = testing::Combine(
-        testing::Combine(
-                testing::ValuesIn(inputShapesBS2_4D),
-                testing::ValuesIn(inputPrecisions),
-                testing::ValuesIn(depthToSpaceModes),
-                testing::Values(1, 2),
-                testing::Values(CommonTestUtils::DEVICE_CPU)),
-        testing::ValuesIn(filterCPUInfoForDevice(CPUParamsBS2_4D))
-);
-
-INSTANTIATE_TEST_SUITE_P(smoke_CPUDepthToSpaceBS2_4D, DepthToSpaceLayerCPUTest, depthToSpaceBS2_4DParams, DepthToSpaceLayerCPUTest::getTestCaseName);
+INSTANTIATE_TEST_SUITE_P(smoke_CPUDepthToSpaceBS2_4D, DepthToSpaceLayerCPUTest,
+                         testing::Combine(
+                                 testing::ValuesIn(ov::test::static_shapes_to_test_representation(inputShapesBS2_4D)),
```

Comment:
NIT. Here you use `ov::test::` namespace resolution, but below not, please align the code to be consistent.